src/daos/sql-query-builder.php

```diff
@@ -63,14 +67,20 @@ public function joins( $join_statement ) {
 
   /**
    * For setting a limit on the query other than the default
-   * @param $new_limit : integer
+   *
+   * @param $new_limit : intger | string
    */
   public function limit( $new_limit ) {
-    $this->limit = intval( $new_limit );
+    if ( false === stripos( $new_limit . '', ':' ) ) {
```

Comment:
it would be clearer if we used starts_with() https://github.com/gios-asu/nectary/blob/develop/src/utilities/string-utilities.php#L7